rpc/query_server.go

```diff
@@ -1133,6 +1125,21 @@ func (s *QueryServer) getBlockHeightFromHash(hash []byte) (uint64, error) {
 	}
 }
 
+func (s *QueryServer) getEthAccount(address eth.Data) (loom.Address, error) {
+	snapshot := s.StateProvider.ReadOnlyState()
+	defer snapshot.Release()
```

Comment:
The snapshot should be passed in as a parameter so we don't create multiple snapshots to handle a single query.